### DISCUSSION TITLE
Guild Vs Ambitus

### DISCUSSION THREAD

**Discussion by StevenDickens** [zopencommunity/meta#30] [Q&A] [answered]
This looks like the exact same scope as the Open Mainframe Project Ambitus project?

https://www.openmainframeproject.org/all-projects/ambitus

Why 2 different attempts to do the same thing?

**Comment by MikeFultonDev** [comment]
i responded in the other question you have - do you want to merge these?

**Reply by StevenDickens** [reply]
I think one project with the objective of porting Open Source to z/OS is the way to go, and Ambitus seems to have already got some traction.  Also the optics look better if OMP does it rather than IBM try and lead this effort.

**Reply by StevenDickens** [reply]
Mike, basically have the IBM team run under Ambitus and achieve the same objective, but make it look like a wider community effort rather than an IBM project... Also you get to use the OMP infrastructure...

**Comment by MikeFultonDev** [comment, marked as answer]
Since @jbostian responded in: https://github.com/ZOSOpenTools/meta/discussions/29 I am going to recommend we close this discussion.
